examples/website/carto-sql/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server --progress --hot --open",
+    "start-local": "webpack-dev-server --env.local --progress --hot --open"
+  },
+  "dependencies": {
+    "@deck.gl/core": "^8.2.8",
```

Comment:
`deck.gl/react` is missing. Test locally and make sure that `yarn start` works.

You can simply use `"deck.gl": "^8.3.0-beta"` as the dependency. It will bring in all required packages.